spec/requests/api/v3/providers/suggest_provider_spec.rb

```diff
@@ -68,7 +70,8 @@
 
   it "limits responses to a maximum of 10 items" do
     11.times do
-      create(:provider, provider_name: "provider X")
+      courses = [build(:course, site_statuses: [build(:site_status, :findable)])]
```

Comment:
`[build(:site_status, :findable)]` can probably be DRYed a bit

Reply:
not too sure what you mean...

site_status, comes from site, which is needed for provider